libs/damap/src/lib/widgets/license-wizard/license-wizard-list.ts

```diff
@@ -174,7 +174,7 @@ export const agpl1 = {
   name: 'license.wizard.license.agpl1.name',
   priority: 50,
   available: false,
-  url: '',
+  url: 'https://www.affero.org/oagpl.html',
```

Comment:
This does not lead to a license site (site is not maintained).
No expert on this, but it may have been superseded by AGPLv3 ? @ZenoLC Do you know anything about this?

Reply:
let us remove this URL from here and from the backend and then move forward. The PRs look fine otherwise.